agents/check_mk_agent.linux

```diff
@@ -977,39 +998,21 @@ section_mailqueue() {
             for queue_dir in ${multi_instances_dirs}; do
                 if [ -n "${queue_dir}" ]; then
                     postfix_queue_dir=$(postconf -c "${queue_dir}" 2>/dev/null | grep ^queue_directory | sed 's/.*=[[:space:]]*//g')
-                    read_postfix_queue_dirs "${postfix_queue_dir}" "${queue_dir}"
+                    postfix_instance_name=$(postconf -c "${queue_dir}" -h multi_instance_name 2>/dev/null)
+                    read_postfix_queue_dirs "${postfix_queue_dir}" "${postfix_instance_name}"
```

Comment:
you are right, i mentioned it in my PR already, but before i change my patch i want to have a decision which solution to choose.
First i think in "postfix_mailq" and "postfix_mailq_status" the service name should be both the same, because they are belong together. 

So there are two scenarios:
1. Choose the configuration directory ($queue_dir)
    postfix_mailq: Service name not changes
    postfix_mailq_status: Service name definitiv changes, because before it was a single word now its a path
2. Choose the instance name which is choosen by the administrators of the mail system (my current choice)
    postfix_mailq: Service name definitv changes, because before it was a path now its a single word
    postfix_mailq_status: Service name may change, if the last part of path is different to instance_name

For me the instance_name better represents the postfix processes. The instance name is often shorter then the path to the configuration files, both are unique.
The change of the service name only happens if you have a multi instance setup, on most systems there are only a single instance setup an no change in the service names.
Maybe a complete rewrite of the postfix_mailq plugin would be better, because postfix_mailq has performance data and
postfix_mailq_status only looks if the master process is present, they can be merged, so only one service is needed.
But that would be a incompatible change too.

So please make a decision an tell me which name you prefer for postfix_mailq and which for postfix_mailq_status.
Or you prefer if i file a bug report on your support portal?

Reply:
We prefer if the service names are not changed. A change in service name will mean you loose the history for that service.

Reply:
Done the requested change